lib/shouldi/having.ex

```diff
@@ -54,7 +54,11 @@ defmodule ShouldI.Having do
         shouldi_path = unquote(var)[:shouldi_having_path] || []
```

Comment:
When I set `var \\ quote(do: context)`,

- the following `setup` in `having` context succeeded :+1: 
    - in this case, `var` was `{:context, [], ShouldI.Having}`
```elixir
setup do
  {:ok, [context: :setup_in_having]}
end
```

- But the following `setup _` is failed because of `unbound variable _` error
```elixir
setup _ do
  :ok
end
```